database/db.js

```diff
@@ -0,0 +1,42 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  user: 'root',
+  password: '',
+  database: 'rewhy'
+});
+
+// connection.connect(function(err) {
+//   if (err) {
+//     console.log('err--------', err)
+//   } else {
+//     console.log('SUCCESSFUL CONNECTION to db!');
+//   }
+// })
+
+
```

Comment:
Looks like you migrated your code here over to your schema and seed files? Might want to clean this file out.